cmd/extractDiffs.go

```diff
@@ -64,3 +78,14 @@ func extractDiffs() {
 		LogWithCommand.Fatalf("extracting diffs failed: %s", err.Error())
 	}
 }
+
+func createFilterQuery() ethereum.FilterQuery {
```

Comment:
Would love to get these addresses from the config file, but wasn't sure the best way to handle that because a lot of our interaction with the config file is in  vdb-mcd-transformers.

Reply:
This seems legit to me. If we wanted to do it via config, maybe we could require that it be structured as:
```
[contract]
    [contract.NAME]
        address  = "0x00"
        abi      = '[{}]`
        deployed = 0
```
and then read from it the same way we do in `cmd/execute`